packages/steps/test/lion-steps.test.js

```diff
@@ -27,91 +27,101 @@ async function checkWorkflow(steps, expected) {
 }
 
 describe('lion-steps', () => {
+  it('has "steps" getter that returns default slot elements', async () => {
+    const el = await fixture(html`
+      <lion-steps>
+        <lion-step>Step 1</lion-step>
+        <lion-step initial-step>Step 2</lion-step>
+        <other-step-element>Step 3</other-step-element>
+        <steps-extension-element slot="new-slot">e.g. steps indicator</steps-extension-element>
+      </lion-steps>
+    `);
```

Comment:
@maxevilmind this is a new light DOM API which we support now. If you want your new component to be in Light DOM, it must be a different slot. While default slot can be definition only have step elements (`<lion-step>` or it's extensions)

but you also have an option to not use Light DOM of `<lion-steps>` at all and put your extra feature as a sibling of lion-steps (even before this MR is merge)

unfortunately other elements inside the default slot will not be supported